e2e/scripts/update_shannon_config_fallback_eth.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
```

Comment:
A `ci` sub-directory here seems like a good option, i.e. `e2e/scripts/ci`, to contain all the CI-only scripts.